src/components/__tests__/FlawForm.spec.ts

```diff
@@ -614,6 +623,83 @@ describe('FlawForm', () => {
     const disabledOptions = sourceField.findAll('option[hidden]');
     expect(disabledOptions.length).toBe(0);
   });
+
+  it('should show all affects in sorting order', async () => {
+    const flaw = sampleFlaw();
+    flaw.affects = [
+      {
+        uuid: 'bff95399-ef12-43fe-878d-4629297c2aa8',
+        flaw: '3ede0314-a6c5-4462-bcf3-b034a15cf106',
+        affectedness: 'AFFECTED',
+        resolution: 'FIX',
+        ps_module: 'shell-4',
+        ps_component: 'openshift',
+        ps_product:'Red Hat Enterprise Linux',
+        impact: 'LOW',
+        // cvss2: '',
+        // cvss2_score: null,
+        // cvss3: '',
+        // cvss3_score: null,
```

Comment:
I think if we won't use those we could just omit them. (Same thought for next cases).